src/site/content/en/blog/maskable-icon/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Better PWA icons with maskable icons
+subhead: A new icon format to use adaptive icons on supporting platforms.
+description: |
+  Maskable icons are a new icon format that give you more control and let your Progressive Web App
+  use adaptive icons. By supplying a maskable icon, your icon can look great on all Android
+  devices.
+authors:
+  - tigeroakes
+date: 2019-12-19
+hero: hero.png
```

Comment:
Please run `hero.png` through tinypng.com. I got it down to 200KB. This keeps the size of the repository itself down.